test/PrintAsObjC/classes.swift

```diff
@@ -89,13 +89,13 @@ class ClassWithCustomName2 {}
 class ClassWithCustomNameSub : ClassWithCustomName {}
 
 
-// CHECK-LABEL: @interface ClassWithNSObjectProtocol <NSObject>
+// CHECK-LABEL: @interface ClassWithNSObjectProtocol
 // CHECK-NEXT: @property (nonatomic, readonly, copy) NSString * _Nonnull description;
 // CHECK-NEXT: - (BOOL)conformsToProtocol:(Protocol * _Nonnull)_ SWIFT_WARN_UNUSED_RESULT;
 // CHECK-NEXT: - (BOOL)isKindOfClass:(Class _Nonnull)aClass SWIFT_WARN_UNUSED_RESULT;
 // CHECK-NEXT: - (nonnull instancetype)init OBJC_DESIGNATED_INITIALIZER;
 // CHECK-NEXT: @end
-@objc class ClassWithNSObjectProtocol : NSObjectProtocol {
+@objc class ClassWithNSObjectProtocol {
```

Comment:
This is no longer testing the same thing; please have it conform to something else. (Preferably something where the Objective-C name doesn't match the Swift name.)